CONTRIBUTING.md

```diff
@@ -91,6 +91,8 @@ for how to set up your development environment, or ask in #hackers-test on Disco
 Developing for Flutter
 ----------------------
 
+### Framework & Engine
+
 If you would prefer to write code, you may wish to start with our list of [good first issues](https://github.com/flutter/flutter/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
 
 To develop for Flutter, you will eventually need to become familiar
```

Comment:
hmm, actually, a lot of this section applies in general (e.g. 3, 4, 5, the video, the wiki, etc). Really only 1 and 2 are specific to framework and engine.

Reply:
There are parts of 3 that apply to DevTools (~80% of the overview section on the [wiki](https://github.com/flutter/flutter/wiki/Tree-hygiene), the "Using Git" and "Getting a Code Review" sections), but there are is also quite a bit that is not relevant for DevTools.

We already document the workflow for making changes on the `flutter/devtools` CONTRIBUTING guide (which I've just re-written [here](https://github.com/flutter/devtools/blob/25c78a85ca59e3682cd53e783b980820281473d2/CONTRIBUTING.md)), so I would default to those instructions instead of making the user have to parse all the content on the Flutter wiki and have to decipher what is relevant to DevTools and what is not.

4 _sort of_ applies. The DevTools style guide is [here](https://github.com/flutter/devtools/blob/25c78a85ca59e3682cd53e783b980820281473d2/STYLE.md), which is also linked to from our own CONTRIBUTING guide.

5 does apply to DevTools as well.

The video mostly applies to DevTools, except for the parts that are specific to contributing `flutter/flutter`

The medium article applies, except for the section about not using dartfmt - we do use the Dart formatter in DevTools.